tests/integration/components/user-sidebar-test.js

```diff
@@ -1,22 +1,29 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 import startMirage from '../../helpers/setup-mirage-for-integration';
+import PageObject from 'ember-cli-page-object';
+import userSidebarComponent from '../../pages/components/user-sidebar';
+
+let page = PageObject.create(userSidebarComponent);
 
 moduleForComponent('user-sidebar', 'Integration | Component | user sidebar', {
   integration: true,
   setup() {
     startMirage(this.container);
   },
+  beforeEach() {
+    page.setContext(this);
+  },
   afterEach() {
     server.shutdown();
   }
 });
 
 test('it renders', function(assert) {
   assert.expect(1);
-  this.render(hbs`{{user-sidebar}}`);
+  page.render(hbs`{{user-sidebar}}`);
 
-  assert.equal(this.$('.user-sidebar').length, 1, 'Component\'s element is rendered');
+  assert.equal(page.userSidebarCount, 1, 'Component\'s element is rendered');
```

Comment:
We ought to be removing the "it renders" tests altogether, so feel free to remove this one.